apis/adbc/README.md

```diff
@@ -0,0 +1,38 @@
+# ADBC API Cookbook for Spice.ai
```

Comment:
Need to add to README.md in root too. Also don't forget to update: https://github.com/spiceai/docs/blob/trunk/website/src/pages/cookbook/index.mdx